AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -107,7 +107,7 @@ + (BOOL)tableExists:(NSString *)tableName inOpenedDatabase:(void *)db {
 
 + (NSUInteger)versionInOpenedDatabase:(void *)db {
   NSArray *result = [MSDBStorage executeSelectionQuery:@"PRAGMA user_version" inOpenedDatabase:db];
-  return [result[0][0] unsignedIntegerValue];
+  return (result.count > 0) ? [result[0][0] unsignedIntegerValue] : 0;
```

Comment:
You don't need to check result[0] count?

Reply:
No I made sure it's never empty ;)